src/constants.py

```diff
@@ -248,6 +248,9 @@
     "H2": {UNIT: "kWh_eleq/kgH2", VALUE: 32.87},
 }
 
+# dict keys in results_json file
+TIMESERIES = "timeseries"
```

Comment:
keys in json files are defined in `constants_json_string.py` module ;) and this one exists already in there (you just need to import it from there